### DISCUSSION TITLE
NUT non si avvia

### DISCUSSION THREAD

**Discussion by Moxser** [indomus/forum#1012] [HUB: Home Assistant]
Ciao a tutti

Poco tempo fa ho acquistato un UPS Eaton 3S.
Installato correttamente e funzionante fino a una settimana fa con una buona e funzionante automazione per lo spegnimento del server in caso di caduta della corrente elettrica.
E' collegato a una porta USB del Raspberry ma non funziona più e non so proprio capire quale sia la causa.

Vi incollo qui l'errore durante l'avvio del Addon NUT e la mia configurazione.

![image](https://user-images.githubusercontent.com/72693930/140602059-bc0dc24e-e7b4-4764-a889-3fd2c794d467.png)
![image](https://user-images.githubusercontent.com/72693930/140602085-a668b5f9-5293-4c13-956e-d1ff196ca0d3.png)
![image](https://user-images.githubusercontent.com/72693930/140602256-20018f90-3532-41c5-b35b-549ca08e455c.png)

Sembra che il driver non si avvii ma non so proprio come poter ovviare al problema.
Spero che qualcuno mi possa aiutare perché non ho fatto nessun tipo di modifica ha semplicemente smesso di funzionare.
Ho già provato a riavviare, cambiare USB ecc.

**Comment by snakuzzo** [comment]
Scusa...ma perché stai usando un addon ?
L'integrazione NUT è prevista nativamente. Leggi [QUI](https://www.home-assistant.io/integrations/nut/)

**Reply by Moxser** [reply]
Ciao.
Ero convinto di dover usare un Addon come da vostra guida su indomus, comunque tutto funzionava benissimo fino a una settimana fa quindi con addon e integrazione.
Ora ho disabilitato addon ma in integrazioni il risultato è questo

![image](https://user-images.githubusercontent.com/72693930/140607406-a1813802-3571-416b-b1c6-09b704a4231d.png)
Hai qualche suggerimento in merito?

**Reply by Moxser** [reply]
Come da guida ho:
- collegato l'UPS tramite porta USB al raspberry
- Installato (tramite supervisor) e configurato i “Network UPS Tools” (driver corretto - utente - porta - password ecc)
- avviato il servizio (tutto funzionava)
- poi da integrazioni ho cercato network ups tools - mi ha trovato il mio UPS, ho inserito le credenziali e tutto filava liscio. 
Una settimana fa ha iniziato a darmi gli errori riportati sopra.

Cosa faccio disinstallo tutto e parto da 0? 
Ci sono procedimenti differenti?

**Reply by snakuzzo** [reply]
Aspetta...forse sto confondendo io le cose.
L'integrazione nativa ti permette di collegarti al servizio nut già presente
L'addon invece "immagino" che tiri su un container con il servizio nut per la gestione dell'ups.

Nel tuo caso quindi, diversamente da quanto detto prima, devi usare l'addon

**Reply by Moxser** [reply]
Infatti è proprio quello che ho fatto e che ora non funziona più.

**Reply by snakuzzo** [reply]
Quando qualcosa non va tocca leggere i log.
Inutile basarsi su quello che vedi sulla gui e dire "non va più"

Nel tuo caso vai a leggere i log di
addon
supervisor
core

**Reply by Moxser** [reply]
Allora 
Questo è il log dell'addon:
```
Failed to to call /addons/a0d7b954_nut/stats - Container addon_a0d7b954_nut is not running
```

Log Supervisor:
```
21-11-06 19:35:13 ERROR (SyncWorker_4) [supervisor.docker.interface] Container addon_a0d7b954_nut is not running
```

Log Core:
```
2021-11-06 19:35:13 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /addons/a0d7b954_nut/stats - Container addon_a0d7b954_nut is not running
```

Poi notavo che all'interno delle voci dell'HOST mi da questa info
```
usb 1-1.4: usbfs: USBDEVFS_CONTROL failed cmd usbhid-ups rqt 128 rq 6 len 255 ret -110
```

Non so se può essere utile.

**Comment by KC-inDomus** [comment]
secondo me sbagli driver.

**Reply by Moxser** [reply]
Ciao KC.
Il driver è quello consigliato direttamente dal sito Networkupstools per questo modello di UPS e che in effetti è quello che ho utilizzato da due mesi a questa parte senza nessun problema. 

![image](https://user-images.githubusercontent.com/72693930/141011347-a4576574-48fd-4532-a37c-bb456a99b244.jpeg) 

Dici che dovrei usare qualche altro driver? Se si quale? 
Oppure tramite terminale c'è la possibilità di verificare o aggiornare il driver?

**Reply by bubez81** [reply]
Era successo anche a me con un apc...un bug dell'ups stesso...per risolvere ho dovuto togliere batteria aspettare un po', rimetterla ed è ripartito. In pratica lo devi resettare. Controlla la procedura del tuo

**Reply by Moxser** [reply]
Questo fine settimana faccio un tentativo.
Avevo pure io il dubbio che potesse essere un problema dell'UPS stesso ma avendolo acquistato 2 mesi fa non pensavo potesse già avere dei problemi.
In ogni caso le batterie sono sostituibili e c'è una guida su come sostituirle quindi immagino che togliendole e aspettando una decina di minuti i condensatori interni si dovrebbero scaricare e si dovrebbe resettare da solo.

**Reply by bubez81** [reply]
ma non è un problema di batteria sul mio (apc) è un bug software
Si comunque quella procedura dovrebbe essere universale

**Reply by Moxser** [reply]
Si si chiaro non intendevo un problema di batteria ma per fare un reset sono costretto a toglierle.
Sulle istruzioni non ci sono indicazioni di reset software, ameno che non mi collego con un PC tramite USB all'UPS e vedo se con il loro programma riesco a capire se posso resettarlo da li senza togliere le batterie.

**Reply by bubez81** [reply]
mah togliendo la batteria la vedo più semplice....non dovrebbe essere difficile farlo
